### PR TITLE
upgrade: docusaurusV2

### PR DESCRIPTION
**Pull request by KishaniKandasamy**
Fixes: #[Add issue number here]

Changes:
upgrade: docusaurusV2

Screenshots of the change:
Screenshot (26)

### CODE REVIEW

**Comment by juzhiyuan** [comment]
@guoqqqi Please have a try to review this PR.

File: docs/committer-guide.md [hunk 1]
```diff
@@ -2,28 +2,27 @@
...
+6.  Improve the website.
 
-7. Write document or article.
+7.  Write document or article.
 
-8. Any form of contribution that is not mentioned above.
+8.  Any form of contribution that is not mentioned above.
 
-If you would like to contribute, please send an email to dev@apisix.apache.org to let us know!
+If you would like to contribute, please send an email to [dev@apisix.apache.org](mailto:dev@apisix.apache.org) to let us know!
```
Comment (guoqqqi) [inline docs/committer-guide.md]: A space after the list number is better

File: docs/2fa.md [hunk 2]
```diff
@@ -2,7 +2,6 @@
 id: 2fa
 title: 2FA
 ---
-
 ## Two-factor authentication(2FA)
```
Comment (guoqqqi) [inline docs/2fa.md:5]: `</br>`  are needed to keep the content clear and easy to read

File: docs/contributor-guide.md [hunk 3]
```diff
@@ -28,66 +27,76 @@ Please fee free to report bugs, submit suggestions, or submit PRs according to t
 Fork the Apache APISIX repo to your own repo to work, then setting proper upstream.
 
 ```sh
+
 git remote add upstream https://github.com/apache/apisix.git
+
 ```
```
Comment (guoqqqi) [inline docs/contributor-guide.md:33]: Extra  `</br>`

File: docs/contributor-guide.md [hunk 4]
```diff
@@ -28,66 +27,76 @@ Please fee free to report bugs, submit suggestions, or submit PRs according to t
...
 
 Switch to your forked master branch, pull codes from upstream, then create a new branch.
 
 ```sh
+
 $ git checkout master
 $ git pull upstream master
 $ git checkout -b IssueNo
+
 ```
```
Comment (guoqqqi) [inline docs/contributor-guide.md:53]: No need `</br>`

File: docs/contributor-guide.md [hunk 5]
```diff
@@ -28,66 +27,76 @@ Please fee free to report bugs, submit suggestions, or submit PRs according to t
...
-2. Then push codes to your fork repo.
+2.  Then push codes to your fork repo.
 
 ```sh
+
 $ git add modified-file-names
 $ git commit -m 'commit message'
 $ git push origin issueNo
+
 ```
```
Comment (guoqqqi) [inline docs/contributor-guide.md:69]: As before

File: docs/download.md [hunk 6]
```diff
@@ -41,6 +40,7 @@ Find all APISIX releases in the [Archive repository](https://archive.apache.org/
 It is essential that you verify the integrity of the downloaded files using the PGP or SHA signatures. The PGP signatures can be verified using GPG or PGP. Please download the KEYS as well as the asc signature files for relevant distribution. It is recommended to get these files from the main distribution directory and not from the mirrors.
 
 ```sh
+
 gpg -i KEYS
```
Comment (guoqqqi) [inline docs/download.md:44]: As before

File: docs/download.md [hunk 7]
```diff
@@ -50,11 +50,13 @@ pgpk -a KEYS
 # or
 
 pgp -ka KEYS
+
```
Comment (guoqqqi) [inline docs/download.md:53]: As before

File: docs/release-guide.md [hunk 8]
```diff
@@ -44,12 +43,14 @@ Once a version is released, it will enter the maintenance mode and will only acc
 
 ### Install GPG
 
-Download GnuPG from https://gnupg.org/download/index.html. There are differences between the 1.x and 2.x versions of the GnuPG commands, and the following descriptions are based on the GnuPG 2.x versions.
+Download GnuPG from https:. There are differences between the 1.x and 2.x versions of the GnuPG commands, and the following descriptions are based on the GnuPG 2.x versions.
```
Comment (guoqqqi) [inline docs/release-guide.md:46]: miss "https: "

File: docs/release-guide.md [hunk 9]
```diff
@@ -44,12 +43,14 @@ Once a version is released, it will enter the maintenance mode and will only acc
...
 
 After the installation is complete, execute the following command to check the version.
 
 ```sh
-$ gpg --version
+
```
Comment (guoqqqi) [inline docs/release-guide.md:51]: As before

File: docs/release-guide.md [hunk 10]
```diff
@@ -59,102 +60,67 @@ Execute the following command to create the key:
...
-Email address: ${Enter your email address}
-Comment: ${Enter comments}
-You selected this USER-ID:
-  "${Entered username} (${Entered comments}) &lt;${Entered email address}>"
-
-Change (N)ame, (C)omment, (E)mail or (O)kay/(Q)uit? O
-You need a Passphrase to protect your secret key. # Enter passphrase
+
+
+
```
Comment (guoqqqi) [inline docs/release-guide.md:83]: The whole content is lost

File: docs/release-guide.md [hunk 11]
```diff
@@ -59,102 +60,67 @@ Execute the following command to create the key:
...
-------------------------------
-pub 4096R/30B5FD72 2020-01-02
-uid Yuansheng Wang <membphis@apache.org>
-sub 4096R/3D2F913D 2020-01-02
+
+
 ```
 
-And `30B5FD72` is the ID of pub key
+And `` is the ID of pub key
```
Comment (guoqqqi) [inline docs/release-guide.md:102]: The previous content is missing

File: docs/release-guide.md [hunk 12]
```diff
@@ -59,102 +60,67 @@ Execute the following command to create the key:
...
-gpg: sending key 30B5FD72 to hkp server pool.sks-keyservers.net
+
+
 ```
 
-`pool.sks-keyservers.net` is one of the pub key servers, the pub key will be automatically synchronized between each server, just choose anyone.
+`` is one of the pub key servers, the pub key will be automatically synchronized between each server, just choose anyone.
 
-### Add `Key Fingerprint` to id.apache.org
+### Add `` to id.apache.org
```
Comment (guoqqqi) [inline docs/release-guide.md:116]: missing

File: docs/release-guide.md [hunk 13]
```diff
@@ -59,102 +60,67 @@ Execute the following command to create the key:
...
-
-/home/resty/.gnupg/pubring.gpg
-------------------------------
-pub 4096R/30B5FD72 2020-01-02
-Key fingerprint = 0F91 BE0A 55A7 B22F DE1A CEEC 3352 48FD 30B5 FD72
-uid Yuansheng Wang <membphis@apache.org>
-sub 4096R/3D2F913D 2020-01-02
+
+
+
```
Comment (guoqqqi) [inline docs/release-guide.md:123]: missing

File: docs/team.md [hunk 14]
```diff
@@ -30,7 +29,7 @@ title: Team
 
 ### Committer
 
-> The following list excludes PMC members above.
+&gt; The following list excludes PMC members above.
```
Comment (guoqqqi) [inline docs/team.md:32]: `>` Here stands for reference.

File: docs/team.md [hunk 15]
```diff
@@ -55,10 +54,10 @@ title: Team
 
 ## Contributors
 
-> You can find all contributors of Apache APISIX from GitHub contribution list.
+&gt; You can find all contributors of Apache APISIX from GitHub contribution list.
```
Comment (guoqqqi) [inline docs/team.md:57]: should ues `>`

File: website/.docusaurus/codeTranslations.json [hunk 16]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (guoqqqi) [inline website/.docusaurus/codeTranslations.json:1]: the end need a new line

File: website/.docusaurus/docusaurus-plugin-content-blog/default/blog-c06.json [hunk 17]
```diff
@@ -0,0 +1,11 @@
...
+  "permalink": "/blog",
+  "page": 1,
+  "postsPerPage": 10,
+  "totalPages": 1,
+  "totalCount": 2,
+  "previousPage": null,
+  "nextPage": null,
+  "blogDescription": "Blog",
+  "blogTitle": "Blog"
+}
```
Comment (guoqqqi) [inline website/.docusaurus/docusaurus-plugin-content-blog/default/blog-c06.json:11]: the end need a new line

File: website/.docusaurus/docusaurus-plugin-content-blog/default/blog-post-list-prop-default.json [hunk 18]
```diff
@@ -0,0 +1,13 @@
...
+    {
+      "title": "初探 Kubernetes Service APIs",
+      "permalink": "/blog/2020/12/18/a-first-look-at-kubernetes-service-api"
+    },
+    {
+      "title": "Envoy and Apache APISIX: Another way to implement the Envoy filter",
+      "permalink": "/blog/2020/12/16/another-way-to-implement-envoy-filter"
+    }
+  ]
+}
```
Comment (guoqqqi) [inline website/.docusaurus/docusaurus-plugin-content-blog/default/blog-post-list-prop-default.json:13]: also

File: website/.docusaurus/docusaurus-plugin-content-blog/default/site-blog-2020-08-22-new-website-md-4e6.json [hunk 19]
```diff
@@ -0,0 +1,14 @@
...
+  "date": "2020-08-22T00:00:00.000Z",
+  "tags": [],
+  "title": "New website for Apache APISIX",
+  "readingTime": 0.13,
+  "truncated": true,
+  "prevItem": {
+    "title": "Envoy and Apache APISIX: Another way to implement the Envoy filter",
+    "permalink": "/blog/2020/12/16/another-way-to-implement-envoy-filter"
+  }
+}
```
Comment (guoqqqi) [inline website/.docusaurus/docusaurus-plugin-content-blog/default/site-blog-2020-08-22-new-website-md-4e6.json:14]: As before

File: website/.docusaurus/routesChunkNames.json [hunk 20]
```diff
@@ -0,0 +1,98 @@
...
+  },
+  "/docs/security-852": {
+    "component": "component---theme-doc-item-178-a40",
+    "content": "content---docs-security-291-43d"
+  },
+  "/docs/team-c9f": {
+    "component": "component---theme-doc-item-178-a40",
+    "content": "content---docs-team-525-f54"
+  }
+}
```
Comment (guoqqqi) [inline website/.docusaurus/routesChunkNames.json:98]: all files  need a new line at the end

File: docs/release-guide.md [hunk 21]
```diff
@@ -15,26 +14,26 @@ The entire flow is comprised of the following four phases.
 
 This phase will decide if a feature is ready to be released as well as the release time.
 
-- A new target milestone will be created in Github. (e.g. [`2.3`](https://github.com/apache/apisix-dashboard/milestone/6)).
-- A discussion will be started on dev mailing list [dev@apisix.apache.org](dev@apisix.apache.org) for gathering ideas for the next release.
-- Maintainer team will then mark the issues and pull requests with the target milestone.
+-   A new target milestone will be created in Github. (e.g. [`2.3`](https://github.com/apache/apisix-dashboard/milestone/6)).
+-   A discussion will be started on dev mailing list [dev@apisix.apache.org](dev@apisix.apache.org) for gathering ideas for the next release.
+-   Maintainer team will then mark the issues and pull requests with the target milestone.
```
Comment (guoqqqi) [inline docs/release-guide.md:19]: As before

File: docs/release-guide.md [hunk 22]
```diff
@@ -164,89 +130,41 @@ sub 4096R/3D2F913D 2020-01-02
...
-$ svn --username=${Apache username} commit -m "release 1.0"
-Adding         1.0
-Adding  (bin)  1.0/apache-apisix-1.0-src.tar.gz
-Adding  (bin)  1.0/apache-apisix-1.0-src.tar.gz.asc
-Adding         1.0/apache-apisix-1.0-src.tar.gz.sha512
-Transmitting file data ...
-Committed revision 37435.
+
+
+
```
Comment (guoqqqi) [inline docs/release-guide.md:167]: missing

File: docs/release-guide.md [hunk 23]
```diff
@@ -255,16 +173,16 @@ Click [here](https://lists.apache.org/thread.html/rf534952a6b2d23ed6efdd61f15b40
 
 ### Send VOTE RESULT thread to the dev mailing list
 
-Click [here](https://lists.apache.org/thread.html/r9153da737a4590dbbba7272acc004cf4bc7abefa488069810d790643%40%3Cdev.apisix.apache.org%3E) to view the reference email at least 3 `+1` [binding votes](https://www.apache.org/foundation/voting.html#binding-votes) is required, then send the vote result to dev@apisix.apache.org.
+Click [here](https://lists.apache.org/thread.html/r9153da737a4590dbbba7272acc004cf4bc7abefa488069810d790643%40%3Cdev.apisix.apache.org%3E) to view the reference email at least 3 `` [binding votes](https://www.apache.org/foundation/voting.html#binding-votes) is required, then send the vote result to [dev@apisix.apache.org](mailto:dev@apisix.apache.org).
 
 ### Move package from dev to dist
 
-Invite [PMCs](https://apisix.apache.org/team/) to move KEYS and package under the [`release`](https://dist.apache.org/repos/dist/release/apisix/) directory.
+Invite [PMCs](https://apisix.apache.org/team/) to move KEYS and package under the [``](https://dist.apache.org/repos/dist/release/apisix/) directory.
```
Comment (guoqqqi) [inline docs/release-guide.md:180]: missing `release`

File: docs/security-guide.md [hunk 24]
```diff
@@ -2,9 +2,8 @@
 id: security
 title: Security
 ---
-
 The Apache Software Foundation takes a rigorous stance on eliminating security issues in its software projects. Apache APISIX is also very concerned Security issues related to its features and functionality.
```
Comment (guoqqqi) [inline docs/security-guide.md:5]: `</br>`

File: docs/subscribe-guide.md [hunk 25]
```diff
@@ -2,13 +2,12 @@
...
+2.  Receive and reply the confirmation e-mail. After Step 1, you will receive a confirmation e-mail from [dev-help@apisix.apache.org](mailto:dev-help@apisix.apache.org) (if you can not receive that, please check “RSS feeds”, “junk e-mail” or other items). Reply that e-mail directly or click the link in e-mail to reply, with any subjects or contents.
 
-3. Receive the Welcome e-mail. After finishing the two steps above, an e-mail with the subject of WELCOME to dev@apisix.apache.org will be sent to your e-mail address. Until now, you have succeeded in subscribing Apache APISIX mailing-list.
+3.  Receive the Welcome e-mail. After finishing the two steps above, an e-mail with the subject of WELCOME to [dev@apisix.apache.org](mailto:dev@apisix.apache.org) will be sent to your e-mail address. Until now, you have succeeded in subscribing Apache APISIX mailing-list.
 
-4. Until now, you can interact with community by your subscribed email or track email conversations by Archived email list.
+4.  Until now, you can interact with community by your subscribed email or track email conversations by Archived email list.
```
Comment (guoqqqi) [inline docs/subscribe-guide.md:13]: a space

File: website/docusaurus.config.js [hunk 26]
```diff
@@ -0,0 +1,120 @@
...
+      "copyright": "Copyright © 2019-2021 The Apache Software Foundation. Apache APISIX, APISIX™, Apache, the Apache feather logo, and the Apache APISIX project logo are either registered trademarks or trademarks of the Apache Software Foundation.",
+      
+    },
+    "algolia": {
+      "apiKey": "79e72fedcf3719ba85c552f710ade8a3",
+      "appId": "ZHVP417Y1Y",
+      "indexName": "apache-apisix-website"
+    }
+  }
+}
```
Comment (guoqqqi) [inline website/docusaurus.config.js]: need `br`

File: docs/committer-guide.md (hunk 1 above)
Comment (KishaniKandasamy) [reply docs/committer-guide.md]: Hi,These things were changed when running automated migration  using  CLI.

**Comment by KishaniKandasamy** [comment]
Hi  , The changes  which were committed with  the 1 st commit ,  are done  when running automated migration using CLI.

**Comment by KishaniKandasamy** [comment]
> Hi , The changes which were committed with the 1 st commit , are done when running automated migration using CLI.

Screenshot (31)

**Comment by guoqqqi** [comment]
> > Hi , The changes which were committed with the 1 st commit , are done when running automated migration using CLI.
> 
> 
> 
> Screenshot (31)
> 
> 

I see. It seems that these Markdown files are better migrated by hand

File: website/.docusaurus/client-modules.js [hunk 27]
```diff
@@ -0,0 +1,8 @@
+export default [
+  require("E:\\Final\\apisix-website\\website\\node_modules\\remark-admonitions\\styles\\infima.css"),
+  require("E:\\Final\\apisix-website\\website\\node_modules\\remark-admonitions\\styles\\infima.css"),
+  require("E:\\Final\\apisix-website\\website\\node_modules\\remark-admonitions\\styles\\infima.css"),
+  require("E:\\Final\\apisix-website\\website\\node_modules\\infima\\dist\\css\\default\\default.css"),
+  require("E:\\Final\\apisix-website\\website\\node_modules\\@docusaurus\\theme-classic\\lib\\prism-include-languages"),
+  require("E:\\Final\\apisix-website\\website\\static\\css\\custom.css"),
+];
```
Comment (guoqqqi) [inline website/.docusaurus/client-modules.js:8]: I'm not sure it's compatible with both Windows and Mac OS？

File: website/.docusaurus/docusaurus.config.js [hunk 28]
```diff
@@ -0,0 +1,214 @@
...
+        "docs": {
+          "showLastUpdateAuthor": true,
+          "showLastUpdateTime": true,
+          "path": "..\\docs",
+          "sidebarPath": "..\\website1.0\\sidebars.json"
+        },
+        "theme": {
+          "customCss": [
+            "E:\\Final\\apisix-website\\website\\static\\css\\custom.css"
+          ]
```
Comment (guoqqqi) [inline website/.docusaurus/docusaurus.config.js:64]: Is it normal to write like this on Mac OS？

**Comment by KishaniKandasamy** [comment]
> > > Hi , The changes which were committed with the 1 st commit , are done when running automated migration using CLI.
> > 
> > 
> > Screenshot (31)
> 
> I see. It seems that these Markdown files are better migrated by hand

Cool.I will upgrade docusaurus 2 mannually and submit the PR ASAP.

**Pull request closed without merging**